src/back/Tile.js

```diff
@@ -31,7 +31,7 @@ Tile.prototype.render = function (project, map, cb) {
     this.setupBounds();
     map.zoomToBox(this.projection.forward([this.minX, this.minY, this.maxX, this.maxY]));
     var im = new mapnik.Image(this.height, this.width);
-    map.render(im, {scale: project.mml.scale}, cb);
+    map.render(im, {scale: project.mml.scale, variables: {zoom: this.z}}, cb);
```

Comment:
I vaguely remember an unresolved by where `variables` was not handled in the vector tile pathway. Create a ticket a node-mapnik if you see this. Can't recall if I fixed it...